components/service-operator/internal/aws/sdk/sdkfakes/fake_client_happy.go

```diff
@@ -176,5 +177,63 @@ func NewHappyClient(outputs map[string]string) *FakeClient {
 		}
 	}
 
+	client.DescribeImagesPagesWithContextStub = func(context.Context, input *ecr.DescribeImagesInput, fn func(*ecr.DescribeImagesOutput, bool) bool, ...request.Option) error {
```

Comment:
these function signatures look funny with some named and some not named args .... can we do one or the other and either name all of them or name none of them